### PR TITLE
Convert agent docs to use Jekyll

### PR DESCRIPTION
**Pull request by sarah256**
This change converts the agent docs to be populated with Markdown, using Jekyll

To run locally, run `jekyll serve` in the docs directory

### CODE REVIEW

File: docs/.jekyll-cache/Jekyll/Cache/Jekyll--Cache/b7/9606fb3afea5bd1609ed40b622142f1c98125abcfe89a76a661b0e8e343910 [hunk 1]
```diff
@@ -0,0 +1 @@
...e"=>false, "include"=>[".htaccess"], "exclude"=>[".sass-cache", ".jekyll-cache", "gemfiles", "Gemfile", "Gemfile.lock", "node_modules", "vendor/bundle/", "vendor/cache/", "vendor/gems/", "vendor/ruby/"], "keep_files"=>[".git", ".svn"], "encoding"=>"utf-8", "markdown_ext"=>"markdown,mkdown,mkdn,mkd,md", "strict_front_matter"=>false, "show_drafts"=>nil, "limit_posts"=>0, "future"=>false, "unpublished"=>false, "whitelist"=>[], "plugins"=>[], "markdown"=>"kramdown", "highlighter"=>"rouge", "lsi"=>false, "excerpt_separator"=>"\n\n", "incremental"=>false, "detach"=>false, "port"=>"4000", "host"=>"127.0.0.1", "baseurl"=>nil, "show_dir_listing"=>false, "permalink"=>"date", "paginate_path"=>"/page:num", "timezone"=>nil, "quiet"=>false, "verbose"=>false, "defaults"=>[], "liquid"=>{"error_mode"=>"warn", "strict_filters"=>false, "strict_variables"=>false}, "kramdown"=>{"auto_ids"=>true, "toc_levels"=>[1, 2, 3, 4, 5, 6], "entity_output"=>"as_char", "smart_quotes"=>"lsquo,rsquo,ldquo,rdquo", "input"=>"GFM", "hard_wrap"=>false, "guess_lang"=>true, "footnote_nr"=>1, "show_warnings"=>false}, "name"=>"JF Agent Documentation", "livereload_port"=>35729, "serving"=>true, "watch"=>true, "url"=>nil}:ET
```
Comment (ametzger) [inline docs/.jekyll-cache/Jekyll/Cache/Jekyll--Cache/b7/9606fb3afea5bd1609ed40b622142f1c98125abcfe89a76a661b0e8e343910:1]: I think we want to add the `.jekyll-cache` directory needs to be added to `.gitignore`

File: docs/_layouts/basic-page.html [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+        $(document).ready(async function() {
+            // Get index.html content
+            var templates = document.createElement('template');
+            const response = await fetch('index.html');
+            const result = await response.text();
+            templates.innerHTML = result;
+
+            // Get nav bar template from index.html and put it on the page
+            var template = templates.content.querySelector('#navTemplate'); 
+            console.log(template);
```
Comment (ametzger) [inline docs/_layouts/basic-page.html:59]: errant `console.log`?

File: docs/_site/usage-modes.html [hunk 3]
```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
```
Comment (ametzger) [inline docs/_site/usage-modes.html:1]: Should the `_site` directory be checked in or will github automatically build the HTML from the markdown?

Comment (sarah256) [reply docs/_site/usage-modes.html]: Good catch, looks like GH will do it for us

Comment (sarah256) [reply docs/_site/usage-modes.html]: Should this be on the `.gitignore` as well then?

Comment (ametzger) [reply docs/_site/usage-modes.html]: I think so, yeah

File: docs/jira-setup-guide.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+This is where you'll add and populate your environmental variables file for Jira.  If you want your Git instance to work with the agent, see the setup guide corresponding to the type of Git instance you have - it will include Jira instructions as well.
+
+
+## Instructions
+
+1. Add the following section to your environment variable file. This is the same file mentioned [in step 2](env-var-setup-guide.html). Adding the following variables allows the agent to access your Jira data:
+    <p class="code-block"><code>
+        JIRA_USERNAME=...<br/>
+        JIRA_PASSWORD=...
+    </code></p>
```
Comment (ametzger) [inline docs/jira-setup-guide.md:23]: one more quick question: does this come out properly in the HTML or should it be converted into markdown style code blocks? e.g.

    ```
    JIRA_USERNAME=...
    JIRA_PASSWORD=...
    ```

Comment (sarah256) [reply docs/jira-setup-guide.md:23]: Here is what it looks like without these tags

Screen Shot 2021-07-29 at 11 56 43 AM

Comment (sarah256) [reply docs/jira-setup-guide.md:23]: The longer code blocks are probably a better example
Screen Shot 2021-07-29 at 12 00 36 PM

**Review by ametzger — APPROVED** [review]
looks gr8

**Pull request merged**